PhysicsTools/NanoAOD/plugins/MuonVtxAgnosticIsoProducer.cc

```diff
@@ -0,0 +1,158 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include <eigen3/Eigen/Dense>
+#include <eigen3/unsupported/Eigen/CXX11/Tensor>
+#include "ROOT/RVec.hxx"
+#include <Math/Vector4D.h>
+#include "TVector2.h"
+
+namespace deltas {
```

Comment:
[DataFormats/Math/interface/deltaR.h](https://github.com/cms-sw/cmssw/blob/93e2e3051588c8f931cb559201d76f9edafdb9ec/DataFormats/Math/interface/deltaR.h)

[DataFormats/Math/interface/deltaPhi.h](https://github.com/cms-sw/cmssw/blob/93e2e3051588c8f931cb559201d76f9edafdb9ec/DataFormats/Math/interface/deltaPhi.h)

Reply:
Do I have to acknowledge/implement this comment if we decide to just merge the PR and move on?